baking/src/tezos_baking/tezos_setup_wizard.py

```diff
@@ -182,6 +182,23 @@ def get_node_version_hash():
     return get_proc_output("octez-node --version").stdout.decode("ascii").split()[0]
 
 
+def get_node_version():
+    version = get_proc_output("octez-node --version").stdout.decode("ascii")
+    major_version, minor_version = (
+        re.search(
+            r"\([0-9]+\.[0-9]+\)",
+            version,
+        )
+        .group()
+        .strip("()")
+        .split(".")
+    )
+    return int(major_version), int(minor_version)
```

Comment:
`minor_version` seems to be unused below since we still use the old approach with hashes

I'd suggest removing the old logic and using the `minor_version` too for consistency